src/libs/ajax/DatasetMetrics..js

```diff
@@ -0,0 +1,11 @@
+import { Config } from '../config';
+import { getApiUrl, fetchOk } from '../ajax';
+
+
+export const DatasetMetrics = {
```

Comment:
This was originally `Metrics` but that term was overloaded with the API calls to Bard, so I renamed it. If there is a more accurate name, please let me know.